### PR TITLE
Proposal: Add MIT license

### PR DESCRIPTION
**Pull request by nqthqn**
Should we put multiple names (primary contributors?) in the Copyright section?

Like Go:

https://github.com/golang/go/blob/master/AUTHORS
https://github.com/golang/go/blob/master/CONTRIBUTORS
https://github.com/golang/go/blob/master/LICENSE
...

### CODE REVIEW

**Comment by nqthqn** [comment]
@champagneabuelo We forgot to review PRs at todays meetup. Hopefully we can do that on Monday :)

**Comment by champagneabuelo** [comment]
Ah yeah, we can definitely do that on Monday! Maybe that can be a part of the Openboard hack schedule: go over any open PRs and discuss them among the group.

**Comment by nqthqn** [comment]
@daved per our "need 2 sign offs" rule, I've added you as a reviewer :)

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 The Open Board Authors
```
Comment (daved) [inline LICENSE]: I think this should be "Openboard", but am not sure.

**Pull request merged**